scripts/limsquery.py

```diff
@@ -0,0 +1,24 @@
+
+from genologics.entities import *
+from genologics.lims import *
+from genologics.config import BASEURI, USERNAME, PASSWORD
+
+
+
+
+lims= Lims(BASEURI, USERNAME, PASSWORD)
+
+
+def  get_udfs():
+
+    step_name='End repair, A-tailing and adapter ligation (TruSeq RNA) 4.0'
```

Comment:
You could change that to whichever step you want to get udfs for.

On Tue, Oct 20, 2015 at 3:41 PM, senthil10 <notifications@github.com> wrote:

> In scripts/limsquery.py
> <https://github.com/SciLifeLab/genologics/pull/134#discussion_r42495803>:
>
> > @@ -0,0 +1,24 @@
> > +
> > +from genologics.entities import *
> > +from genologics.lims import *
> > +from genologics.config import BASEURI, USERNAME, PASSWORD
> > +
> > +
> > +
> > +
> > +lims= Lims(BASEURI, USERNAME, PASSWORD)
> > +
> > +
> > +def  get_udfs():
> > +
> > +    step_name='End repair, A-tailing and adapter ligation (TruSeq RNA) 4.0'
>
> so this script is only for this step ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SciLifeLab/genologics/pull/134/files#r42495803>.
>